pkg/storage/mongo/mongo.go

```diff
@@ -9,14 +9,14 @@ import (
 	"strings"
 	"time"
 
-	"github.com/globalsign/mgo"
 	"github.com/gomods/athens/pkg/config"
 	"github.com/gomods/athens/pkg/errors"
+	"github.com/mongodb/mongo-go-driver/mongo"
 )
 
 // ModuleStore represents a mongo backed storage backend.
 type ModuleStore struct {
-	s        *mgo.Session
+	s        *mongo.NewClient
```

Comment:
@arpitgogia would you mind renaming this struct field name to something more explicit?